src/app/components/MostRead/index.tsx

```diff
@@ -138,11 +137,17 @@ const MostRead = ({
     isBff,
   });
 
-  const eventTrackingData = {
-    ...blockLevelEventTrackingData,
-    ...(sendOptimizelyEvents && {
-      optimizely,
+  const mergedEventTrackingData: EventTrackingData & {
+    optimizely?: ReactSDKClient | null | undefined;
+  } = {
+    componentName: eventTrackingData?.componentName ?? 'most-read',
+    ...(eventTrackingData?.groupTracker && {
+      groupTracker: eventTrackingData.groupTracker,
+    }),
+    ...(eventTrackingData?.itemTracker && {
+      itemTracker: eventTrackingData.itemTracker,
```

Comment:
Is itemTracker ever defined at this point?

I see it being given its properties at the lower level of item https://github.com/bbc/simorgh/blob/7ab7fd9571cfeae1874a408e2393097acdc8dce1/src/app/components/MostRead/Canonical/Item/index.tsx#L54-L66